### DISCUSSION TITLE
Is it possible to disable specific metrics in prometheus exporter?

### DISCUSSION THREAD

**Discussion by feigenblatt** [rabbitmq/rabbitmq-server#2739] [Questions] [answered]
Hi, 

I'm using RabbitMQ 3.8.9 and the Prometheus plugin.
I'd like to know if it's possible to disable some of the metrics and if there's any example on how to do it.
The reason I ask is that I need to enable per object metrics to get queue specific data and the response is getting large, but there are lots of Erlang metrics I'm not using at this time.

Thanks

**Comment by michaelklishin** [comment, marked as answer]
There is no way to disable some metrics but there is a separate API endpoint for per-object metrics as of `3.8.10`, see release notes. It can be used occasionally for some specific requests while the main endpoint uses aggregation.

**Reply by michaelklishin** [reply]
So the endpoint is `GET {hostname}:15692/metrics/per-object` instead of the standard `{hostname}:15692/metrics/` which follows the [aggregation settings](https://www.rabbitmq.com/prometheus.html#metric-aggregation). Available in `3.8.10`, `3.8.11` and so on.
